### DISCUSSION TITLE
Client work

### DISCUSSION THREAD

**Discussion by ugintl** [JulianPrieber/littlelink-custom#148] [General]
Should I tell my client that I am using littlelink custom or I should just customize and setup the app for them? They do not need to know everything!

**Comment by JulianPrieber** [comment]
I mean, that choice is pretty much up to you.

They're bound to find out at some point anyway, so I see no reason in withholding that information.

I'd recommend you give people access to the [documentation](https://littlelink-custom.com/docs/d/about-littlelink-custom/) in case someone wants to change anything.
Also, I'd say giving people access to the [theme page](https://littlelink-custom.com/themes.php) is a very important part of this project.

**Reply by ugintl** [reply]
This is what I thought
